packages/cli/src/command-implementations/project-command.ts

```diff
@@ -44,15 +50,49 @@ export abstract class ProjectCommand<C extends ProjectCommandDefinition> extends
     return new utils.cache.FSKeyValueCache<ProjectCache>(this.projectPaths.abs.projectCacheFile)
   }
 
-  protected parseBot(bot: bpsdk.Bot) {
+  protected async prepareBotIntegrationInstances(bot: bpsdk.Bot, api: ApiClient) {
+    const integrationList = _(bot.props.integrations).values().filter(utils.guards.is.defined).value()
+
+    const { apiInstances, localInstances } = this._splitApiAndLocalIntegrationInstances(integrationList)
+
+    const fetchedInstances: ApiIntegrationInstance[] = await bluebird.map(localInstances, async (instance) => {
```

Comment:
I'm curious, is there a reason bluebird's `map()` function was preferred over using the native `Promise.map()`?

Reply:
Does Promise.map really exists ? I know `Promise.all()` but it behaves differently, it accepts an array of promise instead of an async mapping function like:
```ts
const myPromises = localInstances.map(async (instance) => {
    // ...
    return stuff
})
const fetchedInstances: ApiIntegrationInstance[] = await Promise.all(myPromises)
```

I find `bluebird.map` to be clearer